### PR TITLE
Fix SDK generation when package version is > 9.x.x

### PR DESCRIPTION
**Pull request by gquadrati**
- Bug fixing: when `@pagopa/openapi-codegen-ts` package version is > 9.x.x, regex expression fails to load it.
- Upgrade test code with latest libraries version.

### CODE REVIEW

File: templates/client-sdk-publish/template.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ steps:
     PACKAGE=${{ parameters.generatorPackageName }}
     # If the application use a specific version of the generator package, use it; otherwise, just use latest version from the registry
     # Warning: yarn only is supported
-    CODEGEN_VERSION=$(yarn list --depth=0 | grep $PACKAGE |  grep -Eo "([0-9]\.)+[0-9]+" || echo '*')
+    CODEGEN_VERSION=$(yarn list --depth=0 | grep $PACKAGE |  grep -Eo "([0-9]*\.)+([0-9]*)" || echo '*')
```
Comment (gquadrati) [inline templates/client-sdk-publish/template.yaml:59]: the core of the PR is this line of code :)

Comment (balanza) [reply templates/client-sdk-publish/template.yaml:59]: I'm trying to figure out the new regex. Do we have an example of the issue we're fixing with such change? I think we may just miss a `+` after the first `[0-9]` (that is: it wasn't supporting packages with major version > 9)

Comment (balanza) [reply templates/client-sdk-publish/template.yaml:59]: Which makes sense, as the error arose just as we realeased `@pagopa/openapi-codegen-ts@10`

Comment (balanza) [reply templates/client-sdk-publish/template.yaml:59]: What about this:
```
([0-9]+\.)+[0-9]+
```

Comment (gquadrati) [reply templates/client-sdk-publish/template.yaml]: Sure, fine for me.. done in [3aab484](https://github.com/pagopa/azure-pipeline-templates/pull/85/commits/3aab484637c91ef72530e6e3d5a90dc91fd6f9cd)

**Pull request merged**